README.md

```diff
@@ -10,6 +10,17 @@ To run all specs locally and review code coverage, execute the following command
 npm test -- --watchAll --ci --coverage
 ```
 
+## Authentication in development 
+
+### Login with Apple
+
+The `REACT_APP_APPLE_REDIRECT_URI` environment variable will need to be set to the proxy URL for your `:3006` frontend. This could be the HTTPS URL for your frontend app tunneled via NGROK, for example. 
+
+When you set this environment variable, you will need to ensure that the redirect URL 
+
+- Tunnels to your app running on `localhost:3006`
+- Is added to the list of website URLs for your App Service. You can edit the app service [here](https://developer.apple.com/account/resources/identifiers/serviceId/edit/H5993XZFJ4)
```

Comment:
This is setup against my Apple developer account so you won't be able to look at it - but I wanted to include the link to the resource so at least we have a record of it in our docs